federatedscope/core/configs/cfg_data.py

```diff
@@ -10,15 +10,18 @@ def extend_data_cfg(cfg):
 
     cfg.data.root = 'data'
     cfg.data.type = 'toy'
+    cfg.data.args = ''
+    cfg.data.splitter = ''
+    cfg.data.splitter_args = ''
```

Comment:
maybe we should add some annotations here to show the usage and some alternative options for `splitter` and `splitter_args`